roles/sro_deploy_art_bundle/files/builder_script.yml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: ci-artifacts
+  name: ci-artifacts-script
+  namespace: ci-artifacts
+data:
+  run_index_builder.sh: |
+    #! /bin/bash
+    set -x
+    set -o nounset
+
+    IMAGE_NAME="brew.registry.redhat.io/rh-osbs/openshift-ose-special-resource-operator-bundle"
+
+    BUNDLES="/tmp/bundles"
+    podman search --authfile /var/lib/kubelet/config.json \
+      ${IMAGE_NAME} \
+      --list-tags --limit 500 | sort -r > ${BUNDLES}
+
+    git clone -b ${GIT_COMMIT} ${GIT_REPO} /tmp/special-resource-operator
+    cd /tmp/special-resource-operator
+    COMMITS=`git log --pretty=format:"%h"`
+
+    for commit in ${COMMITS}; do
+      match=`egrep -m 1 ".+\.${commit}\..+" ${BUNDLES}`
+      if [[ -n ${match} ]]; then
+        break
+      fi
+    done
+    if [[ -z ${match} ]]; then
+      echo "No bundles found in registry"
+      exit 1
+    fi
+    BUNDLE=`echo ${match} | awk '{print $1":"$2}'`
+
+    echo "BUNDLE=${BUNDLE}"
+    echo "CATALOG_INDEX=${CATALOG_INDEX}"
+    echo "COMMIT=${commit}"
+
+    REGISTRY_AUTH_FILE=/var/lib/kubelet/config.json /usr/local/go/bin/opm index add --bundles ${BUNDLE} --tag ${CATALOG_INDEX} -c podman
```

Comment:
Cant use the same auth file for podman push because thats an internal registry using service account credentials. I placed the file existing check, though.

Reply:
what I meant (just a suggestion) is:
1. pod spec:
```
   - mountPath: /var/lib/kubelet/config.json
      name: node-pullsecrets
      readOnly: true

    env:
      - name: NODE_PULLSECRET
        value: /var/lib/kubelet/config.json
```
2 . `podman`
```
BUNDLES="/tmp/bundles"
podman search --authfile $NODE_PULLSECRET
```
3. `opm`
```
REGISTRY_AUTH_FILE=$NODE_PULLSECRET /usr/local/go/bin/opm index add --bundles ${BUNDLE} --tag ${CATALOG_INDEX} -c podman
```

or just call the env `REGISTRY_AUTH_FILE`, instead of `NODE_PULLSECRET`
anyway, it's mostly bikeshedding, do as you prefer :)